### PR TITLE
[REF] mail: `ChatWindowHiddenMenu`, review and simplify scss

### PR DESCRIPTION
**Pull request by Brieuc-brd**
Part of the overall v16 SCSS optimization/restyle, task-2704984

task-2781992

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Brieuc-brd** [comment]
@stefanorigano 👋 
This PR is ready for review 👍

File: addons/mail/static/src/components/chat_window_hidden_menu/chat_window_hidden_menu.xml [hunk 1]
```diff
@@ -2,26 +2,26 @@
...
+            <div class="o_ChatWindowHiddenMenu_dropdownToggle dropdown-toggle justify-content-center align-items-center flex-grow-1 d-flex mw-100" t-att-class="{ 'show opacity-50': messaging.chatWindowManager.isHiddenMenuOpen }" t-on-click="_onClickToggle">
+                <div class="o_ChatWindowHiddenMenu_dropdownToggleIcon o_ChatWindowHiddenMenu_dropdownToggleItem me-1 fa fa-comments-o"/>
+                <div class="o_ChatWindowHiddenMenu_dropdownToggleItem o_ChatWindowHiddenMenu_windowCounter mx-1 text-truncate">
                     <t t-esc="messaging.chatWindowManager.allOrderedHidden.length"/>
                 </div>
             </div>
-            <ul class="dropdown-menu dropdown-menu-right o_ChatWindowHiddenMenu_list" t-att-class="{ show: messaging.chatWindowManager.isHiddenMenuOpen }" role="menu" t-ref="list">
+            <ul class="o_ChatWindowHiddenMenu_list m-0 p-0 dropdown-menu dropdown-menu-right overflow-auto" t-att-class="{ show: messaging.chatWindowManager.isHiddenMenuOpen }" role="menu" t-ref="list">
```
Comment (stefanorigano) [inline addons/mail/static/src/components/chat_window_hidden_menu/chat_window_hidden_menu.xml:12]: `m-0 p-0` utility-classes should be placed after `dropdown-menu dropdown-menu-right` component classes

**Review by stefanorigano — CHANGES_REQUESTED** [review]
@Brieuc-brd , just a minor remark, but I-it would be better to wait for https://github.com/odoo/odoo/pull/85839 and https://github.com/odoo/odoo/pull/85842 to be merged though, there are several contact points.
Sorry for the headache

**Review by alexkuhn — APPROVED** [review]
LGTM.

Just note that there's actually a bug with chat window hidden menu being in DOM in mobile, when it shouldn't.

Before changes, it was behind the full-screen chat window, so the user could not interact with it. However, with changes here, it's now above chat window so it's visible.

The problem is unrelated to PR, but this just highlights this problem that exists for quite some time. Will make another task to fix this exact issue.

**Comment by alexkuhn** [comment]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo rebase-ff

**Pull request closed without merging**